inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/config/ControllerExceptionHandler.java

```diff
@@ -85,7 +88,7 @@ public Response<String> handleBindExceptionHandler(HttpServletRequest request, B
     @ResponseBody
     @ExceptionHandler(value = HttpMessageConversionException.class)
     public Response<String> handleHttpMessageConversionExceptionHandler(HttpServletRequest request,
-            HttpMessageConversionException e) {
+                                                                        HttpMessageConversionException e) {
```

Comment:
It is not recommended to indent and align parameters.

Reply:
I got it.